data/language/en-GB.txt

```diff
@@ -4468,6 +4468,8 @@ STR_6156    :Name is reserved
 STR_6157    :Console
 STR_6158    :Failed to load this file...{NEWLINE}Incompatible RCTC version: {COMMA16}
 STR_6159    :Smooth Nearest Neighbour
+STR_6160    :Search
```

Comment:
You need to prefix tooltips with `{SMALLFONT}{BLACK}`. Notice how your tooltip uses the large font and is grey?